source/SIL.AppBuilder.Portal/src/lib/components/InternationalizedDataBox.svelte

```diff
@@ -4,19 +4,31 @@
     
 -->
 <script lang="ts">
```

Comment:
FYI, "Internationalized" is probably not needed in the component name.  Everything should internationalized.